packages/js/browserinstance.js

```diff
@@ -435,9 +442,10 @@ class BrowserInstance {
         }
 
         // Create smashtest/screenshots if it doesn't already exist
-        const dir = path.join('smashtest', 'screenshots');
+        const customPath = reporter.setCustomPath();
+        const dir = path.join(customPath, "/screenshots");
```

Comment:
I don't want to use "/" since it's "\\" in Windows systems

Reply:
I am on windows and it works perfectly, the system interprets it in \ instead of /